src/WebJobs.Script.Grpc/Abstractions/WorkerCreateContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public class WorkerCreateContext
+    {
+        public Uri ServerUri { get; set; }
+
+        public WorkerConfig WorkerConfig { get; set; }
+
+        public string WorkerId { get; set; }
```

Comment:
Can we comment what the strong invariants are for WorkerId and RequestId?   For example, are either 1:1 with other items (process id, function invocation request, etc)

Reply:
`RequestId` - for request response messages only, indicating that a response is the explicit 1:1 response for a request, i.e WorkerInitRequest and WorkerInitResponse.

`WorkerId` is 1:1 with process id, but I'd like an identifier for before the process has started and has an OS assigned id.